xdmod_init.py

```diff
@@ -384,6 +385,33 @@ def main():
             with open("/usr/share/xdmod/libraries/rest.php", "w", encoding="utf-8") as file:
                 file.write(rest_php)
 
+        if os.path.isfile("/usr/share/xdmod/libraries/security.php"):
+            """
+                  $cParams["lifetime"],
+                  $cParams["path"],
+                  $cParams['domain'],
+            -     true
+            +     false
+                  );
+            """
+            security_php = None
+            with open("/usr/share/xdmod/libraries/security.php", encoding="utf-8") as file:
+                security_php = file.read()
+                match1_str = re.search(
+                    r"[ \t]+\$cParams\[\"lifetime\"\],[ \t\n\r\f]*\$cParams\[\"path\"\],[ \t\n\r\f]*\$cParams\['domain'\],[ \t\n\r\f]*(true)[ \t\n\r\f]*\);",
+                    security_php,
+                )
+                if match1_str:
+                    match2_str = re.search(r"true", match1_str.group())
+                    if match2_str:
+                        replace_span = (match2_str.span()[0] + match1_str.span()[0], match2_str.span()[1] + match1_str.span()[0])
+                        begining_str = security_php[: replace_span[0]]
+                        after_str = security_php[replace_span[1] :]
+                        security_php = "".join([begining_str, "false", after_str])
+            if security_php:
```

Comment:
This is always going to be `true` (because you set `security_php = file.read()` on line 399, and that isn't guarded by any sort of conditional).

Reply:
ok, I'll take out the conditional.

Reply:
ok, I'll take out the conditional.

Reply:
I understand your concerns.  It made more sense to do it in the xdmod-init as I am modifying other parts of the configuration.

Reply:
I think this is a response to [an earlier comment](https://github.com/CCI-MOC/xdmod-cntr/pull/39#pullrequestreview-1010503285), rather than the comment here about the `if` statement.